app/editor/src/features/content/list-view/components/tool-bar/filter/AdvancedSearchSection.tsx

```diff
@@ -27,14 +30,11 @@ export interface IAdvancedSearchSectionProps {}
  */
 export const AdvancedSearchSection: React.FC<IAdvancedSearchSectionProps> = () => {
   const [{ sourceOptions }] = useLookupOptions();
-  const [
-    { filter: oFilter, filterAdvanced: oFilterAdvanced },
-    { storeFilter, storeFilterAdvanced },
-  ] = useContent();
+  const [{ filter: oFilter, filterAdvanced }, { storeFilter, storeFilterAdvanced }] = useContent();
 
   const [statusOptions] = React.useState(getEnumStringOptions(ContentStatusName));
   const [filter, setFilter] = React.useState(oFilter);
-  const [filterAdvanced, setFilterAdvanced] = React.useState(oFilterAdvanced);
```

Comment:
Removed to only use redux state.  Not certain why we had two levels of state here.